### ISSUE TITLE
Unable to parse the bond graph involving "NHE" C-terminal cap in PDB files generated by LEaP

### ISSUE THREAD

**Issue by yaoyic** [openmm/openmm#4012]
When I used LEaP to generate AMBER-compatible C-terminal caps, the resulting PDB file can't be simulated by OpenMM with any AMBER force field xmls. The error message looks like such:
```
(openmm) $ python vacuum_relax.py 
Traceback (most recent call last):
  File ".../capped_version/vacuum_relax.py", line 5, in <module>
    sys = forcefield.createSystem(pdb.topology)
  File "${CONDA_PREFIX}/lib/python3.9/site-packages/openmm/app/forcefield.py", line 1218, in createSystem
    templateForResidue = self._matchAllResiduesToTemplates(data, topology, residueTemplates, ignoreExternalBonds)
  File "${CONDA_PREFIX}/lib/python3.9/site-packages/openmm/app/forcefield.py", line 1433, in _matchAllResiduesToTemplates
    raise ValueError('No template found for residue %d (%s).  %s  For more information, see https://github.com/openmm/openmm/wiki/Frequently-Asked-Questions#template' % (res.index+1, res.name, _findMatchErrors(self, res)))
ValueError: No template found for residue 17 (GLY).  The set of atoms matches GLY, but the bonds are different.  For more information, see https://github.com/openmm/openmm/wiki/Frequently-Asked-Questions#template
```
Note that residue 17 is the last real residue, residue 18 is "NHE".
"NHE" is the standard name used by AMBER for NH2 termination according to their manual ([here](https://ambermd.org/doc12/Amber19.pdf) page 211 bottom), but can't be recognized by OpenMM. The evidence is the missing bonds when I printed `pdb.topology.bonds()`.

I tracked down the problem to here. My suggestion is to add an alias `alt1="NHE"`.
https://github.com/openmm/openmm/blob/3dc2f12ff20efcb609dfa5508431065d2c98d028/wrappers/python/openmm/app/data/pdbNames.xml#L240

**Comment by peastman** [comment]
Adding it as an alias sounds reasonable.  A PR would be welcome.

Note that although NHE may be what Amber calls it, it's not standard in any sense.  NHE is actually the name for [2-[N-CYCLOHEXYLAMINO]ETHANE SULFONIC ACID](http://ligand-expo.rcsb.org/files/N/NHE/ccif/7twn_NHE_1_A_202__D__D_.ccif).

**Comment by peastman** [comment]
Fixed by #4014.
